malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorFragmentActivity.java

```diff
@@ -411,19 +411,34 @@ public void onClick(View v) {
                 }
                else
                 {
+                    String date = new SimpleDateFormat("dd/MM/yy").format(new Date());
+                    Date curr_date = getDateObj(date);
                     Date departure=getDateObj(departure_formattedate);
                     Date arrival=getDateObj(arrival_formattedate);
+                    long curr_datel = curr_date.getTime();
                     long departurel = departure.getTime();
                     long arrivall = arrival.getTime();
 
-                    if(arrivall>=departurel) {
+                    if(departurel < curr_datel){
+
+                        Toast.makeText(getApplicationContext(),"Departure date cannot be less than current date",Toast.LENGTH_SHORT).show();
+
+                    }
+
+                    else if(arrivall < departurel)
+                    {
+                        Toast.makeText(getApplicationContext(),"Arrival date cannot be before the Departure date.",Toast.LENGTH_SHORT).show();
+                    }
+
+                    else if(arrivall>=departurel)
+
```

Comment:
Remove extra lines and follow the same { } pattern as done elsewhere in the code. Of opening in the same line.